src/App.jsx

```diff
@@ -23,23 +24,10 @@ export const alex = {
   isMarried: false,
 };
 
+export const people = [misha, olya, alex];
+
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    {people.map(human => <Person person={human} />)}
```

Comment:
If you make a map - you should use key
https://uk.reactjs.org/docs/lists-and-keys.html

```suggestion
    {people.map(human => <Person person={human} key={human.name} />)}
```